README.md

```diff
@@ -34,6 +34,7 @@ Run `make` to build `./bin/cloudwatch.so`. Then use with Fluent Bit:
 * `auto_create_group`: Automatically create log groups (and add tags). Valid values are "true" or "false" (case insensitive). Defaults to false. If you use dynamic variables in your log group name, you may need this to be `true`.
 * `new_log_group_tags`: Comma/equal delimited string of tags to include with _auto created_ log groups. Example: `"tag=val,cooltag2=my other value"`
 * `log_retention_days`: If set to a number greater than zero, and newly create log group's retention policy is set to this many days.
+* `extra_user_agen`: Append a string to the user agent to track usage. Example: `extra_user_agent eks_container_insights`
```

Comment:
Missing a "t" at the end of there... but actually I do not want to document this option. Let's make it "hidden" because we don't really want users to use it and configure it on their own. We want to reserve it for our own use in the configurations we vend/control for EKS Container Insights.

Reply:
Got it. Will remove this part